test/CallHandler-test.ts

```diff
@@ -314,9 +315,9 @@ describe('CallHandler', () => {
         // Now set the config option
         SdkConfig.put({
             voip: {
-                obeyAssertedIdentity: true,
+                obey_asserted_identity: true,
             },
-        });
+        } as unknown as IConfigOptions);
```

Comment:
This is a sign that the signature of `SdkConfig::put` is too strict, presumably it should be `Partial<IConfigOptions>` or similar.

Reply:
was actually just using the wrong method, `SdkConfig.add` works as a patch